MapboxDirections/MBRoadClasses.h

```diff
@@ -0,0 +1,10 @@
+typedef NS_OPTIONS(NSUInteger, MBRoadClasses) {
+    
+    MBRoadClassesToll = (1 << 1),
+    
+    MBRoadClassesRestricted = (1 << 2),
+    
+    MBRoadClassesMotorway = (1 << 3),
+    
+    MBRoadClassesFerry = (1 << 4),
```

Comment:
The documentation comment for this value should discuss its relationship to the [`ferry` transport type](https://github.com/mapbox/MapboxDirections.swift/blob/93dabdebb1375e745b4a09912afa2d1d4c669cc1/MapboxDirections/MBRouteStep.swift#L17-L22), and the documentation for that transport type should likewise refer to this road class.